package/src/components/AccountProfileInfo/v1/AccountProfileInfo.js

```diff
@@ -111,7 +117,7 @@ class AccountProfileInfo extends Component {
     const { components: { ProfileImage }, viewer } = this.props;
 
     return (
-      <AccountProfileInfoContainer>
+      <AccountProfileInfoContainer className={this.props.className}>
```

Comment:
`className` should be destructured from `this.props` in all the components

Reply:
I don't see a need to allocate a variable for all of these one-off references. Unless this is a convention in the style guide that must be followed?

Reply:
It's a convention we follow for clarity. Makes it easier to see at a glance which props each method is using. It's not strictly required, and sometimes we don't do it in a small function that only uses one prop, but in a case like this where there are several, it is best to follow the pattern. I doubt it has much of an impact on performance or memory, but if you have any references that show otherwise, please share.

Reply:
Oh no, it won't impact anything. 10 years ago things like this made an impact on memory, but in 2018 it isn't a problem at all. I'll make the change.